bin/htmlproofer

```diff
@@ -71,6 +71,8 @@ Mercenary.program(:htmlproofer) do |p|
 
     options[:cache] = {}
     options[:cache][:timeframe] = opts['timeframe'] unless opts['timeframe'].nil?
+    
+    options[:http_status_ignore] = Array(options[:http_status_ignore]).map(&:to_i)
```

Comment:
Scratch that. @gjtorikian I can't use both VCR and the executable because the executable doesn't share the same Ruby env.

Trust that [this line](https://github.com/gjtorikian/html-proofer/blob/4434b7eecb19d869c59c92e6480661d18866253c/bin/htmlproofer#L32) does what I'm saying it does?